bot/slack/commands/ip_info.py

```diff
@@ -0,0 +1,49 @@
+import netaddr
+import os
+
+from mozdef_util.geo_ip import GeoIP
+
+
+def is_ip(ip):
+    try:
+        netaddr.IPNetwork(ip)
+        return True
+    except Exception:
+        return False
+
+
+def ipLocation(ip):
+    location = ""
+    try:
+        geoip_data_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../../data/GeoLite2-City.mmdb")
+        geoip = GeoIP(geoip_data_dir)
+        geoDict = geoip.lookup_ip(ip)
+        if geoDict is not None:
+            if 'error' in geoDict:
+                return geoDict['error']
+            location = geoDict['country_name']
+            if geoDict['country_code'] in ('US'):
+                if geoDict['metro_code']:
+                    location = location + '/{0}'.format(geoDict['metro_code'])
+    except Exception:
+        location = ""
+    return location
+
+
+class command():
```

Comment:
Class names should always be in `PascalCase` as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#class-names)

Reply:
Fixed in https://github.com/mozilla/MozDef/pull/1018/commits/458723466763d1f51f2879147d4c1c255c4ae221